ethers-solc/src/artifacts/mod.rs

```diff
@@ -285,7 +285,7 @@ impl Default for Settings {
         Self {
             stop_after: None,
             optimizer: Default::default(),
-            metadata: None,
+            metadata: Default::default(),
```

Comment:
this will deviate from `solc`'s defaults (useLiteralContent)

don't think this matters but interested wdyt?

Reply:
@mattsse I agree, this changes is unnecessary